.ci/pytorch/build.sh

```diff
@@ -223,8 +223,10 @@ if [[ "${BUILD_ENVIRONMENT}" != *android* && "${BUILD_ENVIRONMENT}" != *cuda* ]]
   export BUILD_STATIC_RUNTIME_BENCHMARK=ON
 fi
 
+# Workaround for dind-rootless userid mapping (https://github.com/pytorch/ci-infra/issues/96)
 WORKSPACE_ORIGINAL_OWNER_ID=$(stat -c '%u' "/var/lib/jenkins/workspace")
 sudo chown -R jenkins /var/lib/jenkins/workspace
+trap "sudo chown -R $WORKSPACE_ORIGINAL_OWNER_ID /var/lib/jenkins/workspace" EXIT
```

Comment:
We should run the trap command before `sudo chown -R jenkins /var/lib/jenkins/workspace` so that we can ensure it runs even if the chown command fails.

Reply:
I updated the PR to move the trap up one line.